holidays.py

```diff
@@ -5528,3 +5528,60 @@ class RS(Serbia):
 
 class SRB(Serbia):
     pass
+
+
+class Greece(HolidayBase):
+    # https://en.wikipedia.org/wiki/Public_holidays_in_Greece
+
+    def __init__(self, **kwargs):
+        self.country = 'GR'
+        HolidayBase.__init__(self, **kwargs)
+
+    def _populate(self, year):
+
+        eday = easter(year, method=EASTER_ORTHODOX)
+
+        # New Years
+        self[date(year, JAN, 1)] = "Πρωτοχρονιά [New Year's Day]"
+        # Epiphany
+        self[date(year, JAN, 6)] = "Θεοφάνεια [Epiphany]"
+
+        # Clean Monday
+        self[eday - rd(days=48)] = "Καθαρά Δευτέρα [Clean Monday]"
+
+        # Independence Day
+        self[date(year, MAR, 25)] = "Εικοστή Πέμπτη Μαρτίου [Independence Day]"
+
+        # Good Friday
+        self[eday - rd(days=2)] = "Μεγάλη Παρασκευή [Good Friday]"
```

Comment:
This isn't an official holiday, unless you are working at public services.

Reply:
Thanks, fixed.